File: Livro - Módulo Java/2 - Estrutura Sequencial/Exercicio.29.java
// 29 - Faça um algoritmo que leia o ano de nascimento de uma pessoa e o ano atual, calcule e mostre: 
// - A idade dessa pessoa em anos; 
// - A idade dessa pessoa em meses; 
// - A idade dessa pessoa em dias; 
// - A idade dessa pessoa em semanas. 

import java.util.Scanner;

public class CalculoIdade {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite o ano de nascimento: ");
        int anoNascimento = scanner.nextInt();

        System.out.print("Digite o ano atual: ");
        int anoAtual = scanner.nextInt();

        int idadeAnos = anoAtual - anoNascimento;
        int idadeMeses = idadeAnos * 12;
        int idadeDias = idadeAnos * 365;
        int idadeSemanas = idadeAnos * 52;

        System.out.println("Idade em anos: " + idadeAnos + " anos");
        System.out.println("Idade em meses: " + idadeMeses + " meses");
        System.out.println("Idade em dias: " + idadeDias + " dias");
        System.out.println("Idade em semanas: " + idadeSemanas + " semanas");

        scanner.close();
    }
}

File: Livro - Módulo Python/2 - Estrutura Sequencial/Exercicio.29.py
#29. Faça um algoritmo que leia o ano de nascimento de uma pessoa e o ano atual, calcule e mostre:
# - A idade dessa pessoa em anos;
# - A idade dessa pessoa em meses;
# - A idade dessa pessoa em dias;
# - A idade dessa pessoa em semanas.

anoNasc = int(input('Digite seu ano de nascimento: '))
anoAtual = int(input('Digite o ano atual: '))

idadeAnos = anoAtual - anoNasc
idadeMeses = idadeAnos * 12
idadeDias = idadeAnos * 365
idadeSemanas = idadeDias / 7

print(f"A idade em anos é: {idadeAnos} anos")
print(f"A idade em meses é: {idadeMeses} meses")
print(f"A idade em dias é: {idadeDias} dias")
print(f"A idade em semanas é: {idadeSemanas:.0f} semanas")